build/upload_artifacts_gh.sh

```diff
@@ -6,7 +6,7 @@ set -e
 #
 #
 cd dist
-for filename in $(find  -regex ".*\.\(msi\|rpm\|deb\|zip\|tar.gz\)");do
+for filename in $(find . -name "*.msi" -o -name "*.rpm" -o -name "*.deb" -o -name "*.zip" -o -name "*.tar.gz");do
```

Comment:
linux and mac have different `find` versions. This ☝️  works as expected in both